src/base/bittorrent/session.cpp

```diff
@@ -3816,22 +3823,18 @@ bool Session::addMoveTorrentStorageJob(TorrentHandleImpl *torrent, const QString
         if (iter != m_moveStorageQueue.end()) {
             // remove existing inactive job
             m_moveStorageQueue.erase(iter);
+            LogMsg(tr("\"%1\" has cancelled to move storage from \"%2\" to \"%3\".").arg(torrent->name(), currentLocation, iter->path));
         }
     }
 
-    QString currentLocation = torrent->actualStorageLocation();
-    if (!m_moveStorageQueue.isEmpty() && (m_moveStorageQueue.first().torrentHandle == torrentHandle)) {
-        // if there is active job for this torrent consider its target path as current location
-        // of this torrent to prevent creating meaningless job that will do nothing
-        currentLocation = m_moveStorageQueue.first().path;
-    }
-
-    if (QDir {currentLocation} == QDir {newPath})
+    if (QDir {currentLocation} == QDir {newPath}) {
+        LogMsg(tr("\"%1\" has rejected to move storage to \"%2\".").arg(torrent->name(), newPath));
         return false;
+    }
 
     const MoveStorageJob moveStorageJob {torrentHandle, newPath, mode};
     m_moveStorageQueue << moveStorageJob;
-    qDebug("Move storage from \"%s\" to \"%s\" is enqueued.", qUtf8Printable(currentLocation), qUtf8Printable(newPath));
+    LogMsg(tr("\"%1\" has enqueued move storage from \"%2\" to \"%3\".").arg(torrent->name(), currentLocation, newPath));
```

Comment:
Move job has been enqueued for "\"%1\" from \"%2\" to \"%3\".

This one's tricky. I hope i got it right. "job" doesn't sound very user friendly but i can't think of something else.

Reply:
> I hope i got it right. "job" doesn't sound very user friendly

I can omit "job" but it should be done in all related messages. Are you sure it will be better?

Reply:
>I can omit "job"

Nono it wouldn't make sense if it was removed in what i wrote. I guess what i meant is whether there is a better word or sentence.